server/services/interventionsService.test.ts

```diff
@@ -1050,3 +1051,66 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
 })
+
+describe('serializeDeliusServiceUser', () => {
+  const interventionsService = new InterventionsService(config.apis.interventionsService)
+
+  it('transforms a DeliusServiceUser into a format expected by the Interventions Service, removing "expired" disabilities', () => {
+    const currentDisabilityEndDate = new Date()
+    currentDisabilityEndDate.setDate(currentDisabilityEndDate.getDate() + 1)
+
+    const deliusServiceUser = {
+      otherIds: {
+        crn: 'X123456',
+      },
+      offenderProfile: {
+        offenderLanguages: {
+          primaryLanguage: 'English',
+        },
+      },
+      title: 'Mr',
+      firstName: 'Alex',
+      surname: 'River',
+      dateOfBirth: '01/01/1980',
+      gender: 'Male',
+      ethnicity: 'British',
+      religionOrBelief: 'Agnostic',
+      disabilities: [
+        {
+          disabilityType: {
+            description: 'Autism',
+          },
+          endDate: '',
+          notes: 'Some notes',
+          startDate: '2019-01-22',
+        },
+        {
+          disabilityType: {
+            description: 'Sciatica',
+          },
+          endDate: currentDisabilityEndDate.toString(),
+          notes: 'Some notes',
+          startDate: '2020-01-01',
+        },
+        {
+          disabilityType: {
+            description: 'An old disability',
+          },
+          endDate: '2020-01-22',
+          notes: 'Some notes',
+          startDate: '2020-01-22',
+        },
+      ],
+    } as DeliusServiceUser
+
+    const serviceUser = interventionsService.serializeDeliusServiceUser(deliusServiceUser)
+
+    expect(serviceUser.crn).toEqual('X123456')
```

Comment:
Perhaps we could test the whole object instead so that we don't miss out on testing any fields? e.g. `dateOfBirth` is missing here.

Reply:
Yeah, I mainly tested the ones that involve the most wrangling on our side, and left most of the ones that have the same key, but I'll add the rest in.